src/routes/post.route.ts

```diff
@@ -20,6 +20,7 @@ class PostRoute implements Routes {
     this.router.delete(`${this.path}/:post_id`, AuthMiddleware, this.postController.deletePost);
     this.router.get(`${this.path}/:post_id/likes`, AuthMiddleware, this.postController.getLikesFromPost);
     this.router.post(`${this.path}/:post_id/likes`, AuthMiddleware, this.postController.likePost);
+    this.router.delete(`${this.path}/:post_id/:like_id`, AuthMiddleware, this.postController.deletePostLike);
```

Comment:
this doesn't follow rest convention. I think if you're just going to delete the like using userId and postId, then we can remove like_id

Reply:
wouldn't line 20 and 23 use the same path and method then?

Reply:
maybe just /likes?